### PR TITLE
Add filter by country id

### PR DESCRIPTION
**Pull request by lissanday**
https://trello.com/c/jjuSgkQb/150-add-countryid-select-filter-to-all-lists

### CODE REVIEW

File: src/app/utils/form/select.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const findOption = (value, options) => {
-  if (!value) return null
+  // if (!value) return null
```
Comment (sem-yura) [inline src/app/utils/form/select.js]: pls remove it if isn’t used in the code

File: src/components/form/Input_Select.pure.jsx [hunk 2]
```diff
@@ -36,7 +36,7 @@ export const InputSelectPure = ({
           menuPortalTarget={document.body}
           styles={customStyles({ hasIcon: !!Icon })}
           defaultValue
-          value={value === null ? null : findOption(value?.toString(), options)}
+          value={findOption(value?.toString(), options)}
           // value={value}
```
Comment (sem-yura) [inline src/components/form/Input_Select.pure.jsx]: remove pls

Comment (lissanday) [reply src/components/form/Input_Select.pure.jsx]: removed

File: src/views/notices/@Routes/index.jsx [hunk 3]
```diff
@@ -167,7 +167,7 @@ const NoticeRoutes = () => {
             title={t('form.offerMedicalAssistance')}
             expandableRowsComponent={withPriceFree(DefaultNotice)}
             noticeType={70}
-            filters={MedicalAssistanceFilter}
+            filters={BasicFilter}
```
Comment (OlehKhozhai) [inline src/views/notices/@Routes/index.jsx]: after the last changes, we move filters to config file,
please change `MedicalAssistanceFilter => BasicFilter` in this file and resolve the conflict

image

Comment (lissanday) [reply src/views/notices/@Routes/index.jsx]: Thanks. Done

**Review by sem-yura — APPROVED** [review]

**Pull request merged**